packages/opentelemetry-exporter-collector/src/CollectorExporterBase.ts

```diff
@@ -59,6 +60,8 @@ export abstract class CollectorExporterBase<
 
     this.shutdown = this.shutdown.bind(this);
 
+    this._concurrencyLimit = config.concurrencyLimit || Infinity;
```

Comment:
please change it to strict type checking 
```suggestion
    this._concurrencyLimit = typeof config.concurrencyLimit === 'number' ? config.concurrencyLimit : Infinity;
```